### PR TITLE
Microsoft Teams Bidirectional Integration

### PR DESCRIPTION
**Pull request by cs-shivaji-kolape**

### CODE REVIEW

**Review by cs-sujeet-ahirrao — APPROVED** [review]

File: info.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        {
+            "name": "sOARFramework",
+            "type": "solutionpack",
+            "label": "SOAR Framework",
+            "minVersion": null
+        }
+    ],
+    "prerequisite": null,
+    "publisher": "Fortinet",
+    "description": "FortiSOAR for Microsoft Teams application allows teams to leverage the power of FortiSOAR as part of their daily communications.",
```
Comment (cs-suyog-jinturkar) [inline info.json:17]: This needs to be reviewed and enhanced. Please get some inputs from Amit and Abhishek Shukla

File: info.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            {
+                "name": "External Channel"
+            }
+        ],
+        "playbooks": [
+            {
+                "name": "02 - Use Case - FortiSOAR for Microsoft Teams"
+            }
+        ],
+        "globalVariables": [
```
Comment (cs-suyog-jinturkar) [inline info.json]: Why do we need these GVs?

File: playbooks/globalVariables.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "name": "Server_fqhn",
+        "value": "localhost.localdomain",
+        "default_value": "localhost.localdomain"
```
Comment (cs-suyog-jinturkar) [inline playbooks/globalVariables.json]: IMO, this is not required. Please confirm

File: playbooks/02 - Use Case - FortiSOAR for Microsoft Teams/Enrich IP.json [hunk 4]
```diff
@@ -0,0 +1,90 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "Enrich IP",
+    "aliasName": null,
+    "tag": null,
+    "description": "Gets triggered on enrichIP command when used in Microsoft Teams",
```
Comment (cs-suyog-jinturkar) [inline playbooks/02 - Use Case - FortiSOAR for Microsoft Teams/Enrich IP.json]: This PB desc also needs to be modified.
Suggestion: This playbook triggers after hitting an enrichIP command using FortiSOAR for Microsoft Teams Bot

File: info.json (hunk 1 above)
Comment (cs-shivaji-kolape) [reply info.json:17]: Will update this part once documentation done.

File: info.json (hunk 2 above)
Comment (cs-shivaji-kolape) [reply info.json]: Not needed, Removed

File: playbooks/globalVariables.json (hunk 3 above)
Comment (cs-shivaji-kolape) [reply playbooks/globalVariables.json]: Not needed, Removed

File: playbooks/02 - Use Case - FortiSOAR for Microsoft Teams/Enrich IP.json (hunk 4 above)
Comment (cs-shivaji-kolape) [reply playbooks/02 - Use Case - FortiSOAR for Microsoft Teams/Enrich IP.json]: Updated suggested description

File: playbooks/02 - Use Case - FortiSOAR for Microsoft Teams/Enrich IP > Enrichment.json [hunk 5]
```diff
@@ -0,0 +1,569 @@
...
+        }
+    ],
+    "groups": [],
+    "priority": "\/api\/3\/picklists\/2b563c61-ae2c-41c0-a85a-c9709585e3f2",
+    "uuid": "9e99ba9b-1536-444e-8aae-db904485567c",
+    "owners": [],
+    "isPrivate": false,
+    "deletedAt": null,
+    "recordTags": [
+        "bot_enabled"
```
Comment (cs-suyog-jinturkar) [inline playbooks/02 - Use Case - FortiSOAR for Microsoft Teams/Enrich IP > Enrichment.json:567]: referenced also needs to be present as a standard practice. Please check few PBs in SFSP

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Comment by cs-akash-jarad** [comment]
playbooks/02 - Use Case - FortiSOAR for Microsoft Teams/Enrichment.json is this empty file required?

**Pull request merged**